src/sign_workflow/signer_mac.py

```diff
@@ -18,7 +18,7 @@
 
 
 class SignerMac(Signer):
-    ACCEPTED_FILE_TYPES = [".pkg", ".dmg"]
+    ACCEPTED_FILE_TYPES = [".pkg", ".dmg", ".dylib"]
```

Comment:
Does our signer sign this file extension? I haven't signed any of this type of files, want to make sure we tested it before.

Reply:
Yes we do! It is a part of sql ODBC signing part. We tested and released the product too.